documentation/docs/20-core-concepts/30-form-actions.md

```diff
@@ -359,6 +359,8 @@ Without an argument, `use:enhance` will emulate the browser-native behaviour, ju
 - render the nearest `+error` boundary if an error occurs
 - [reset focus](accessibility#Focus-management) to the appropriate element
 
+You can add the [`data-sveltekit-reload`](link-options#data-sveltekit-reload) attribute on the `<form>` or the `<button>` to opt-out of this behavior and instead let the browser handle the form submission.
```

Comment:
If we do add this, we should probably document the same under the link options doc https://github.com/sveltejs/kit/blob/main/documentation/docs/30-advanced/30-link-options.md#data-sveltekit-reload